core/src/labels/label.h

```diff
@@ -42,18 +42,26 @@ class Label {
     };
 
     struct Vertex {
-        Vertex(glm::vec2 pos, glm::vec2 uv, glm::vec3 extrude, uint32_t color, uint32_t stroke = 0)
-            : pos(pos), uv(uv), extrude(extrude), color(color), stroke(stroke) {}
-
-        glm::vec2 pos;
-        glm::vec2 uv;
-        glm::vec3 extrude;
+        Vertex(glm::vec2 pos, glm::vec2 uv, glm::vec2 extrude, uint32_t color, uint32_t stroke = 0)
+            : pos(pos * 4.f), uv(uv),
+              extrude(extrude * 256.f),
+              color(color), stroke(stroke) {}
+
+        glm::i16vec2 pos;
+        glm::i16vec2 uv;
+        glm::i16vec2 extrude;
         uint32_t color;
```

Comment:
It may be beneficial to start 4byte attributes at 4byte boundaries.. on a second look this is also the case here :)